src/edgepi/tc/tc_constants.py

```diff
@@ -262,7 +262,7 @@ class AvgMode(Enum):
     AVG_4 = OpCode(0x20, TCAddresses.CR1_W.value, Masks.CR1_HIGH_MASK.value)  # 4 samples averaged
     AVG_8 = OpCode(0x30, TCAddresses.CR1_W.value, Masks.CR1_HIGH_MASK.value)  # 8 samples averaged
     AVG_16 = OpCode(0x40, TCAddresses.CR1_W.value, Masks.CR1_HIGH_MASK.value)  # 16 samples averaged
-
+    AVG_MASK = 0x70
```

Comment:
Why is this pair different from the rest? Is inconsistency ok in this case?

Reply:
good point and fixed aef1556b20336d5d0c033db5c414ed15cc7184e0